packages/editor-ui/src/stores/collaboration.store.ts

```diff
@@ -14,37 +14,48 @@ export const useCollaborationStore = defineStore(STORES.COLLABORATION, () => {
 	const workflowStore = useWorkflowsStore();
 
 	const usersForWorkflows = ref<ActiveUsersForWorkflows>({});
+	const removeEventListener = ref<(() => void) | null>(null);
 
-	pushStore.addEventListener((event) => {
-		if (event.type === 'activeWorkflowUsersChanged') {
-			const activeWorkflowId = workflowStore.workflowId;
-			if (event.data.workflowId === activeWorkflowId) {
-				usersForWorkflows.value[activeWorkflowId] = event.data.activeUsers;
+	const getUsersForCurrentWorkflow = computed(() => {
+		return usersForWorkflows.value[workflowStore.workflowId];
```

Comment:
I know this is an old code, but can we add `?? []` here so it returns an empty array instead of `undefined` if workflow id is not in `usersForWorkflows`?

Reply:
Good point. Done.